editor/src/clj/editor/compute_program.clj

```diff
@@ -0,0 +1,177 @@
+;; Copyright 2020-2024 The Defold Foundation
+;; Copyright 2014-2020 King
+;; Copyright 2009-2014 Ragnar Svensson, Christian Murray
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;;
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;;
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
+(ns editor.compute-program
+  (:require [dynamo.graph :as g]
+            [editor.build-target :as bt]
+            [editor.code.shader :as code.shader]
+            [editor.defold-project :as project]
+            [editor.graph-util :as gu]
+            [editor.material :as material]
+            [editor.protobuf :as protobuf]
+            [editor.protobuf-forms :as protobuf-forms]
+            [editor.resource :as resource]
+            [editor.resource-node :as resource-node]
+            [editor.validation :as validation]
+            [editor.workspace :as workspace]
+            [util.murmur :as murmur])
+  (:import [com.dynamo.render.proto ComputeProgram$ComputeProgramDesc Material$MaterialDesc$ConstantType Material$MaterialDesc$FilterModeMag Material$MaterialDesc$FilterModeMin Material$MaterialDesc$WrapMode]))
+
+(set! *warn-on-reflection* true)
+
+(def ^:private form-data
```

Comment:
A lof of these form entries are identical to the material form, do we want to merge them?

Reply:
Sounds like a good idea! They do appear very similar.